lib/TabsNew/TabsScrollShadow.js

```diff
@@ -23,7 +23,7 @@ export function TabsScrollShadow({ scrollPosition }) {
     opacity: shouldBottomShadowBeVisible ? 1 : 0
   });
 
-  const initialClassName = 'absolute w-full h-4 -mr-4 pointer-events-none';
+  const initialClassName = 'absolute w-full h-4 -mr-4 pointer-events-none z-50';
```

Comment:
💚  is this the lowest the z-index can be? don't want the shadow going above something we aren't expecting. (tabs are usually fixed so shouldn't be an issue, I just find it good practice to go as low as poss for z-index)

Reply:
Makes sense!